lib/AST/ASTImporter.cpp

```diff
@@ -1270,6 +1278,14 @@ bool ASTNodeImporter::ImportDefinition(EnumDecl *From, EnumDecl *To,
   
   To->startDefinition();
 
+  if (TypedefNameDecl *FromTypedef = From->getTypedefNameForAnonDecl()) {
```

Comment:
Looks like this is exactly the same as lines 1145-1152. Could you put them in a lambda or a standalone function, to avoid repetition?